urbs/model.py

```diff
@@ -238,6 +238,12 @@ def create_model(data, dt=1, timesteps=None, dual=False):
         initialize=m.stor_init_bound.index,
         doc='storages with fixed initial state')
 
+    # storage tuples for storages with given energy to power ratio
+    m.sto_en_to_pow_tuples = pyomo.Set(
```

Comment:
I think the name of the tuples is not a lucky choice. Maybe `sto_ep_ratio_tuples`?

Reply:
This suggestion is applied in the new commit.